### PR TITLE
Add test for speech sample.

### LINKED ISSUE #232: Write tests for speech sample

### PR DESCRIPTION
**Pull request by jerjou**
Fixes #232

### CODE REVIEW

File: speech/api/speechrest.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 # Path to local discovery file
 # [START discovery_doc]
-API_DISCOVERY_FILE = 'speech-discovery_google_rest_v1.json'
+API_DISCOVERY_FILE = __file__.rsplit('/', 1)[0] + '/speech-discovery_google_rest_v1.json'
```
Comment (jonparrott) [inline speech/api/speechrest.py]: Wat

``os.path.join(os.path.dirname(__file__), 'speech-discovery_google_rest_v1.json')``

Comment (jerjou) [inline speech/api/speechrest.py]: Yeah - I originally had that, but then I had to add an import that felt irrelevant to the sample.
*shrug. I guess it's good to do it the Right Way...

File: speech/api/speechrest.py [hunk 2]
```diff
@@ -29,7 +30,8 @@
 
 # Path to local discovery file
 # [START discovery_doc]
-API_DISCOVERY_FILE = 'speech-discovery_google_rest_v1.json'
+API_DISCOVERY_FILE = os.path.join(os.path.dirname(__file__),
```
Comment (jonparrott) [inline speech/api/speechrest.py]: please line break at the first parenthesis, I greatly dislike hanging indents.

**Comment by jonparrott** [comment]
LGTM with last nit. Thank you.

**Pull request merged**